ecl2df/summary.py

```diff
@@ -435,6 +432,49 @@ def df(
         )
         logger.warning("Duplicates: %s", list(dframe.columns[dupes]))
         dframe = dframe.loc[:, ~dframe.columns.duplicated()]
+
+    index_duplicates = dframe.index.duplicated(keep="first")
+    if any(index_duplicates):
```

Comment:
Perhaps this should be separated out to a testable auxiliary function?

Reply:
Done, also wrote some tests (including some additional testdata), though they run on the full `df()` function